homotopy-web/src/app/project.rs

```diff
@@ -33,12 +32,12 @@ pub fn project_view(props: &Props) -> Html {
         }
     }));
     html! {
-        <Drawer title="Project" class="project">
+        <>
```

Comment:
Necessary?

Reply:
I think this one actually is needed because there are a few top elements here but I totally agree for the signature one